dvl_gazebo/CMakeLists.txt

```diff
@@ -0,0 +1,200 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(dvl_gazebo)
+
+find_package(catkin REQUIRED COMPONENTS
+  gazebo_dev
+  uuv_sensor_ros_plugins
+  )
+
+set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS} ${GAZEBO_CXX_FLAGS}") 
+
+# Install all the model files
```

Comment:
Probably cleaner to remove the comments.  

I should do the same in the fls_gazebo project ;)

Reply:
I'm not even sure what everything in the CMake file is doing.  I'll go through and clear out the comments before completing the pull, though.